src/transform.ts

```diff
@@ -139,10 +191,26 @@ export interface LookupTransform {
   default?: string;
 }
 
+export interface Comparator {
```

Comment:
Wonder if naming this SortField or WindowSortField make more sense?
(But not sure if there is another thing already called SortField -- but this is definitely slightly different from `sort` 's SortField(Def?).

Reply:
I made a couple changes and this should be reflective of the Comparator defined here now: https://vega.github.io/vega/docs/types/#Compare

Reply:
Um, I haven't looked how you do it -- but seeing that you mention that you make it like "Compare" in Vega.  I would like to mention that it might be more important to make it more consistent with rest of Vega-Lite language than the underlying Vega.